.gitignore

```diff
@@ -60,6 +60,7 @@ __pycache__/
 # Bazel
 /bazel-*
 .bazeloverwriterc
+MODULE.bazel.lock
```

Comment:
Why don't we track the lockfile?

Reply:
The lockfile format is still a little bit unstable. This will be fixed in Bazel 7.2: 
See https://github.com/bazelbuild/bazel/issues/20369

Reply:
I can commit the lockfile. However, it has some rough edges, including panicking bazel sometimes:

```
bazel mod tidy
 checking cached actions
    Fetching repository @@bazel_tools; starting
FATAL: bazel crashed due to an internal error. Printing stack trace:
java.lang.RuntimeException: Unrecoverable error while evaluating node 'com.google.devtools.build.lib.bazel.bzlmod.BazelDepGraphValue$$Lambda$386/0x000000080141d248@803a9e5' (requested by nodes 'com.google.devtools.build.lib.bazel.bzlmod.BazelModTidyValue$$Lambda$385/0x000000080141cb40@5aab2db1')
        at com.google.devtools.build.skyframe.AbstractParallelEvaluator$Evaluate.run(AbstractParallelEvaluator.java:550)
        at com.google.devtools.build.lib.concurrent.AbstractQueueVisitor$WrappedRunnable.run(AbstractQueueVisitor.java:414)
        at java.base/java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1375)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.NullPointerException: null value in entry: gazelle=null
        at com.google.common.collect.CollectPreconditions.checkEntryNotNull(CollectPreconditions.java:33)
        at com.google.common.collect.ImmutableMapEntry.<init>(ImmutableMapEntry.java:54)
        at com.google.common.collect.ImmutableMap.entryOf(ImmutableMap.java:341)
        at com.google.common.collect.ImmutableMap$Builder.put(ImmutableMap.java:450)
        at com.google.devtools.build.lib.bazel.bzlmod.Module.getRepoMappingWithBazelDepsOnly(Module.java:67)
        at com.google.devtools.build.lib.bazel.bzlmod.BazelDepGraphFunction.getExtensionUsagesById(BazelDepGraphFunction.java:233)
        at com.google.devtools.build.lib.bazel.bzlmod.BazelDepGraphFunction.compute(BazelDepGraphFunction.java:126)
        at com.google.devtools.build.skyframe.AbstractParallelEvaluator$Evaluate.run(AbstractParallelEvaluator.java:461)
        ... 7 more

```

https://github.com/bazelbuild/bazel/issues/22423

And also

https://github.com/bazelbuild/buildtools/pull/1263